sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_patch.py

```diff
@@ -57,22 +58,24 @@ def _parse_schema_properties_dict(response_headers: Mapping[str, Union[str, int]
         "version": int(response_headers["Schema-Version"]),
     }
 
+def _normalize_content_type(content_type: str) -> str:
+    return content_type.replace(" ", "").lower()
```

Comment:
a content type can return as `a vro`

Reply:
i.e. These should be equivalent: application/json;serialization=Avro
vs `
application/json; serialization=Avro